core/src/main/java/nl/inl/blacklab/indexers/config/DocIndexerXPath.java

```diff
@@ -674,19 +705,22 @@ protected String findAnnotationMatches(ConfigAnnotation annotation, String value
                 boolean firstValue = true;
                 while (apValuePath.evalXPath() != -1) {
                     apEvalToString.resetXPath();
-                    String annotValue = apEvalToString.evalXPathToString();
-                    annotValue = processString(annotValue, annotation.getProcess(), null);
-                    int increment = firstValue ? 1 : 0;
-                    annotation(annotation.getName(), annotValue, increment, indexAtPositions);
-                    firstValue = false;
+                    String unprocessedValue = apEvalToString.evalXPathToString();
+                    for (String value : processStringMultipleValues(unprocessedValue, annotation.getProcess(), null)) {
```

Comment:
Dit is toch altijd maar 1 value? (maar split kan soms meerdere waardes opleveren natuurlijk... misschien toch een processStringSingleValue maken die dat in dat geval platslaat tot een gewone string?)